internal/app/machined/pkg/controllers/cri/registry.go

```diff
@@ -0,0 +1,118 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
the file should be `registries_config.go` to match controller name

Reply:
Done.